WindSpigot-Server/src/main/java/me/rastrian/dev/PlayerMap.java

```diff
@@ -16,7 +17,7 @@
 public class PlayerMap {
 
 	private static final int CHUNK_BITS = 5;
-	private final Long2ObjectMap<List<EntityPlayer>> map = new Long2ObjectOpenHashMap<>();
+	private final Map<Long, List<EntityPlayer>> map = Maps.newConcurrentMap();
```

Comment:
Instead of switching to a java concurrent map, you should make a class that’s a concurrent variant of `Long2ObjectOpenHashMap`

Reply:
What benefit would this have? I created the ConcurrentIntHashMap to maintain compatibility with some plugins.

Reply:
FastUtil collections are often overall faster than Java’s